src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -125,6 +125,7 @@ public void ProcessStart_UseShellExecute_OnUnix_OpenMissingFile_DoesNotThrow()
 
         [Theory, InlineData(true), InlineData(false)]
         [OuterLoop("Opens program")]
+        [PlatformSpecific(TestPlatforms.Linux)] // s_allowedProgramsToRun is Linux specific
```

Comment:
This file will compile only on Unix - are PlatformSpecific actually needed here? If yes, then I'm curious if we should add any version specific for unix but non-Linux showing what changed

Reply:
This test is Linux specific because of the s_allowedProgramsToRun.
On other platform, it noops because programToOpen is null.
I'll clean it up further.

Reply:
nvmd, I misread the test. It verifies an open program is available on Linux.